packages/tailwindcss-react-aria-components/src/index.test.js

```diff
@@ -399,6 +407,10 @@ test('variants with prefix', async () => {
     --tw-bg-opacity: 1;
     background-color: rgb(255 0 0 / var(--tw-bg-opacity))
 }
+.rac-expanded\:bg-red[data-expanded] {
```

Comment:
I think you just need to add a usage of `rac-expanded:bg-red` to line 311 for this one to pass. I tried to push to your branch, but didn't have permission.

Reply:
Thanks @reidbarber! It turns out I also needed to add the variant to `nnativeVariantSelectors` in the source